package.json

```diff
@@ -175,7 +175,6 @@
     "clsx": "^1.1.1",
     "copy-to-clipboard": "^3.2.0",
     "d3": "^7.6.1",
-    "d3-curve-circlecorners": "^0.1.6",
```

Comment:
One last thing, could you remove the d3-curve-circlecorners from react-app-env.d.ts